app/App.js

```diff
@@ -86,6 +86,13 @@ export default function App() {
 		}
 	};
 
+	const getUserFromSecureStorage = async () => {
+		let res = await SecureStore.getItemAsync('user');
+		if (res) {
+			setUser(JSON.parse(res));
```

Comment:
Add something to check if JWT is expired and if it is; set user to `null` inside of `if (res) {}`